src/bootstrap/flags.rs

```diff
@@ -199,6 +200,13 @@ To learn more about a subcommand, run `./x.py <subcommand> -h`"
                     "mode describing what file the actual ui output will be compared to",
                     "COMPARE MODE",
                 );
+                opts.optopt(
+                    "",
+                    "force",
```

Comment:
I don't like the force naming here. Could we do something like "force-profile" or maybe "only-check"? Force has stronger connotations (and to an extent, denotation) than I think is warranted here.

Reply:
"only-check" is not good, because the end goal is to support `--force build`, `--force run` and perhaps things like `--force >=build` as well, i.e. both increasing and decreasing the "stage" into both directions for cases when we want more or less testing.

Regarding "force", I'm open to suggestions, I just don't want it to be too long/noisy (preferably one word), because at least I am going to use it all the time.

Reply:
Maybe `--profile`? That matches Cargo's naming of check/test/debug/release, at least.

Reply:
debug/release vs. check/build/test feels rather misleading so I don't like `--profile`.

I had the idea of `--stop check` to indicate "stop after `check`" or `--stop-at check` to be more elaborate.

Reply:
`--stop-at` seems fine to me, I guess. I definitely prefer it to `--force`.

Reply:
@Mark-Simulacrum Are you OK with `--pass check` per https://github.com/rust-lang/rust/pull/61755#discussion_r292999660?

Reply:
Yep, that works for me.